dev.yaml

```diff
@@ -0,0 +1,351 @@
+# Create NDM configmap
```

Comment:
Try to add changes to ndm-operator.yaml, instead of creating a new  yaml file.

Reply:
This is something I'm using during development,  I've attached another container to the daemonset which makes it easy to test the API. I need to make some changes to the Makefile as well. I'll remove all of this(build process) once the code looks good.